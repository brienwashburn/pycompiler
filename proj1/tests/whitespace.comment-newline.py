def f(x):
  if x:  
    print(x)
 		
  #these lines should not generate a NEWLINE 	
  print(x+2)
