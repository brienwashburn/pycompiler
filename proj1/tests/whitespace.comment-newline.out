(KEYWORD def)
(ID "f")
(PUNCT "(")
(ID "x")
(PUNCT ")")
(PUNCT ":")
(NEWLINE)
(INDENT)
(KEYWORD if)
(ID "x")
(PUNCT ":")
(NEWLINE)
(INDENT)
(ID "print")
(PUNCT "(")
(ID "x")
(PUNCT ")")
(NEWLINE)
(DEDENT)
(ID "print")
(PUNCT "(")
(ID "x")
(PUNCT "+")
(LIT 2)
(PUNCT ")")
(NEWLINE)
(DEDENT)
(ENDMARKER)
